Dockerfile

```diff
@@ -10,6 +10,7 @@ RUN apk add --update --virtual build-dependencies build-base linux-headers && \
     make build
 
 FROM alpine
+ADD https://pci-ids.ucw.cz/v2.2/pci.ids.gz /usr/share/misc/pci.ids/
```

Comment:
Lets just install the package "hwdata-pci" instead.
`RUN apk add hwdata-pci`

Reply:
Fixed.

Reply:
What is the equivalent package of hwdata-pci on centos or rhel?

Reply:
@zshi-redhat "hwdata" should work on centos rhel. Looks like there's no hwdata-pci for centos which is a subpackage of hwdata in apk (include both pci.ids, usb.ids db). I suggested hwdata-pci for smaller image size.

Reply:
@ahalim-intel thanks for checking it on centos!  sure, there is no problem using hwdata-pci for alpine image, just make sure that we'll have a equivalent package in RHEL so that it can be easily backported to openshift.